packages/material/scss/grid/_layout.scss

```diff
@@ -51,6 +52,10 @@ $grid-group-indicator-gap: 8px !default;
             font-weight: 700;
         }
 
+        thead > tr:not(:only-child) > th {
```

Comment:
Do we have another option for selecting the desired element?

Reply:
I tested different approaches, for now, this is the only possible selector that provides the desired outcome.

Anyway, I am going to change the logic of the borders in the material grid today in order to maintain the alignment of the header with the table. Thus, I might change the above selector, however, I do believe that I will use it.